src/utils/deploy/hash-files.test.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path')
+
+const test = require('ava')
+
+const { withSiteBuilder } = require('../../../tests/utils/site-builder')
+
+const { DEFAULT_CONCURRENT_HASH } = require('./constants')
+const { hashFiles } = require('./hash-files')
+
+test('Hashes files in a folder', async (t) => {
+  await withSiteBuilder('site-with-override', async (builder) => {
+    builder
+      .withNetlifyToml({ config: { functions: { directory: 'functions' } } })
+      .withEnvFile({ path: '.env.development', env: { TEST: 'FROM_DEV_FILE' } })
+      .withContentFile({
+        path: 'lib/util.js',
+        content: `module.exports = { one: 1 }`,
+      })
+      .withContentFile({
+        path: 'functions/function-1.js',
+        content: `
+const { one } = require('../../lib/util')
+
+module.exports.handler = async () => ({ statusCode: 200, body: one })
+`,
+      })
+      .withContentFile({
```

Comment:
[dust] The site builder has `withFunction`, also it has `withContentFiles` that supports an array

Reply:
I know about `withFunction`, but I wanted to `require` the util file which I wouldn't normally do inside the handler, which is why I've opted for using `withContentFile`.

In https://github.com/netlify/cli/pull/2317/commits/008a66b491b4b5c2aeecfd5c6ac7ae188cebe1e6 I've changed it to `withContentFiles`.